.npmignore

```diff
@@ -1,4 +0,0 @@
-tinder.gif
-index.js
-.babelrc
-webpack.config.js
```

Comment:
Since we have the `files` field in `package.json`, it's not necessary to ignore files. Only the files in specified in the package.json will be included